src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs

```diff
@@ -18,12 +18,11 @@ namespace SFA.DAS.EmployerAccounts.Web.Controllers
     [RoutePrefix("accounts/{HashedAccountId}")]
     public class EmployerAgreementController : BaseController
     {
+        private const int ReviewAgreementLater = 2;
```

Comment:
Similarly to above, this diff came about from my merge conflict resolution. I have since checked the file out at parent branch state, but for some reason GitHub still insists it is a diff.